cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -626,4 +633,166 @@ define([
+
+        renderView = function(groupNames) {
+            var view = new GroupList({collection: createGroups(groupNames || [])}).render();
+            appendSetFixtures(view.el);
+            return view;
+        };
+
+        verifyEditingGroup = function(view) {
+            expect(view.$('.group-configuration-edit')).toExist();
+            expect(view.$(newGroupCss)).not.toExist();
+            expect(view.$(addGroupCss)).not.toExist();
+        };
+
+        beforeEach(function() {
+            TemplateHelpers.installTemplates(
+                ['add-list-item', 'no-content-groups', 'content-group-edit', 'content-group-details']
+            );
+        });
+
+        it('shows a message when no groups are present', function() {
+            expect(renderView().$('.no-group-configurations-content')).toExist();
+        });
+
+        it('can create an initial group', function() {
+            var view = renderView();
+            view.$(newGroupCss).click();
+            verifyEditingGroup(view);
+        });
+
+        it('can render groups', function() {
+            var groupNames = ['Group 1', 'Group 2', 'Group 3'];
+            renderView(groupNames).$('.group-configuration-details').each(function(index) {
+                expect($(this)).toContainText(groupNames[index]);
+            });
+        });
+
+        it('can add another group', function() {
+            var view = renderView(['Test Group']);
+            view.$(addGroupCss).click();
+            verifyEditingGroup(view);
+        });
+
+        it('only edits one form at a time', function() {
+            // Should prevent the user from opening more than one edit
+            // form at a time by removing the add button(s) when
+            // editing a group.
+            var view = renderView();
+            view.collection.add({name: 'Editing Group', editing: true});
+            verifyEditingGroup(view);
+        });
+    });
+
+    describe('GroupItem', function() {
+        var renderView;
+
+        beforeEach(function() {
+            TemplateHelpers.installTemplates(['content-group-edit', 'content-group-details']);
+        });
+
+        renderView = function(groupName) {
+            var group = new GroupModel({name: groupName}),
+                collection = new GroupCollection([group]),
+                view = new GroupItem({model: group}).render();
+            appendSetFixtures(view.el);
+            return view;
+        };
+
+        it('can render a details view', function() {
+            var groupName = 'Test Group',
+                view = renderView(groupName);
+            expect(view.$('.group-configuration-details')).toContainText(groupName);
+        });
+
+        it('can render an edit view', function() {
+            var view = renderView('Test Group');
+            view.model.set({editing: true});
+            expect(view.$('.group-configuration-edit')).toExist();
+        });
+    });
+
+    describe('GroupItemEdit', function() {
```

Comment:
Is there a class called ```GroupItemEdit``` that you are testing? I'd rather use a more descriptive name here, e.g. "Group Item Editor" or "Group Item Modal" or whatever it is that is being tested.